### PR TITLE
remove z-index of sup

### LINKED ISSUE #26725: Example output is not visible
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#GitHub-flavored-markdown

In the Styling text section, Superscript doesn't provide an output text:
Untitled design

### What changes are you suggesting?

It will be easier if we provide an example combined with the normal text, like this:
Screenshot_20230715_090634

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by vats147**
### Why:

Closes: #26725

### What's being changed (if available, include any code snippets, screenshots, or gifs):
`Before:`
image

image

`After:`
image

as you clearly see `superscript` text is not visible beacuse of `z-index:-1`

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

File: components/ui/MarkdownContent/stylesheets/table.scss [hunk 1]
```diff
@@ -67,9 +67,6 @@
       border: 0;
       vertical-align: top;
 
-      sup {
-        z-index: -1;
-      }
```
Comment (hectorsector) [inline components/ui/MarkdownContent/stylesheets/table.scss:72]: This change is good, for internal folks it was added as a result of https://github.com/github/docs-internal/pull/34190, but the bug described there no longer exists so good to remove.

File: content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md [hunk 2]
```diff
@@ -39,8 +39,8 @@ You can indicate emphasis with bold, italic, strikethrough, subscript, or supers
 | Strikethrough | `~~ ~~` | None | `~~This was mistaken text~~` | ~~This was mistaken text~~ |
 | Bold and nested italic | `** **` and `_ _` | None | `**This text is _extremely_ important**` | **This text is _extremely_ important** |
 | All bold and italic | `*** ***` | None | `***All this text is important***` | ***All this text is important*** |
-| Subscript | `<sub> </sub>` | None | `<sub>This is a subscript text</sub>` | <sub>This is a subscript text</sub> |
-| Superscript | `<sup> </sup>` | None | `<sup>This is a superscript text</sup>` | <sup>This is a superscript text</sup> |
+| Subscript | `<sub> </sub>` | None | `<sub>This is a subscript text</sub>` | This is a<sub>subscript</sub>text |
+| Superscript | `<sup> </sup>` | None | `<sup>This is a superscript text</sup>` | This is a <sup>superscript</sup>text |
```
Comment (hectorsector) [inline content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md:43]: Leaving this one for content folks to review.

**Review by hectorsector — DISMISSED** [review]
Good on the engineering side - flagged a content change for review though.

**Review by felicitymay — APPROVED** [review]
Many thanks for this pull request 💖 

I'll re-run the preview deployment and check this looks as expected then this should be ready to merge.

**Comment by felicitymay** [comment]
Changes look great. Thanks @vats147 - much appreciated 💖

**Pull request merged**